src/Pact/Types/Term.hs

```diff
@@ -405,17 +416,21 @@ data Use = Use
   { _uModuleName :: !ModuleName
   , _uModuleHash :: !(Maybe Hash)
   , _uInfo :: !Info
-  } deriving (Eq)
-instance Show Use where
-  show Use {..} = "(use " ++ show _uModuleName ++ maybeDelim " " _uModuleHash ++ ")"
+  } deriving (Eq, Show)
+
+instance Pretty Use where
+  pretty Use{..} =
+    let args = pretty _uModuleName : maybe [] (\mh -> [pretty mh]) _uModuleHash
+    in parensSep $ "use" : args
```

Comment:
Should we space "use" a la "use " so args aren't mangled?

Reply:
`parensSep` inserts a space between all the list elements